usaon_vta_survey/templates/response/data_products.html

```diff
@@ -27,7 +27,13 @@ <h3>Data products</h3>
             <br />
           {% endfor %}
 
-          <a href="#">Add relationship</a>
+          <a href="{{url_for(
+                   'view_response_observing_system_data_product_relationships',
```

Comment:
Thinking about our conversation with Hazel, maybe the route becomes `view_response_object_relationships`, and it looks at the arguments it receives to understand what type of relationship it's dealing with. E.g. if it gets `survey_id` and `data_product_id`, it can know that is a valid type of relationship and handle it. But if it gets `data_product_id` and `sba_id`, it knows that's not valid.